jetty-home/pom.xml

```diff
@@ -876,6 +925,14 @@
       <type>jar</type>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>javadoc</artifactId>
+      <version>${project.version}</version>
+      <classifier>javadoc</classifier>
+      <type>jar</type>
+      <optional>true</optional>
+    </dependency>
```

Comment:
you need to add jetty-documentation as a dependency as well to be sure Maven will calculate it as part of the build plan reactor.
Just run `mvn clean install -pl :jetty-home -am` and you will see it's not build before building `jetty-home` whereas it's needed.